docker/example-deployments/proxy-example/README.md

```diff
@@ -0,0 +1,22 @@
+# Proxy Example
+
+## Running
+
+This is an example deployment of Gaffer using a Proxy store. This uses two REST
+APIs on a Map store with data in and another as a proxy pointing to the other.
```

Comment:
This uses two REST APIs, one contains the data and is backed by a Map Store and the other is a Proxy Store configured to forward queries from a second REST API to this Map Store backed REST.